libs/dal/src/repositories/notification/notification.repository.ts

```diff
@@ -170,7 +169,6 @@ export class NotificationRepository extends BaseRepository<
           _id: null,
           weekly: { $sum: { $cond: [{ $gte: ['$createdAt', weekBefore] }, 1, 0] } },
           monthly: { $sum: { $cond: [{ $gte: ['$createdAt', monthBefore] }, 1, 0] } },
```

Comment:
```suggestion
          monthly: { $sum: 1 },
```

I think we could do something like this in the month now.

Reply:
Good point!